dhcp6c_script.c

```diff
@@ -432,12 +510,15 @@ client6_script(scriptpath, state, optinfo)
 		snprintf(s, elen, "%s_%d=", raw_dhcp_option_str, rawop->opnum);
 		const char * hex = "0123456789abcdef";
 		char * val = (char*)malloc(3);
-		for (int o = 0; o < rawop->datalen; o++) {
+		{
```

Comment:
Assignment in the for loop is a C99 thing and the build doesn't specify C99 and this makes OpenBSD's gcc fork angry.